build/Dockerfile.ci

```diff
@@ -1,31 +1,22 @@
 # CI Dockerfile
 # CI requires a special Dockerfile for use with OpenShift Builds which is what is used to build the image. This requires
 # building the operator from the PR source without using the operator-sdk.
-# NOTE: RUN mkdir <dir> is being used in cases where a directory needs to be created as CI image builds does not seem to
-# be creating the directory as a result of the WORKDIR directive. 
-# build stage for building binaries
 
-# We cannot use a golang image as building the kubelet requires rsync and that is not present plus there is no easy way
-# to install it.
-FROM registry.access.redhat.com/ubi8/ubi-minimal as build
+# build stage for building binaries
+FROM registry.svc.ci.openshift.org/ocp/builder:rhel-8-golang-1.15-openshift-4.7 as build
 LABEL stage=build
-RUN microdnf -y install rsync make git tar findutils diffutils && microdnf clean all
-RUN mkdir /build/
-
-# Install go 1.14
-RUN mkdir /build/go
-WORKDIR /build/go
-RUN curl https://golang.org/dl/go1.14.7.linux-amd64.tar.gz -L -o go.tar.gz
-# Check sha256
-RUN echo "4a7fa60f323ee1416a4b1425aefc37ea359e9d64df19c326a58953a97ad41ea5  go.tar.gz" |sha256sum -c
-RUN tar -C /usr/local -xzf go.tar.gz
-ENV GOBIN=/usr/local/go/bin
-ENV PATH=${PATH}:${GOBIN}
+WORKDIR /build/
+
+# Download client binaries
+# TODO: Remove this, see https://issues.redhat.com/browse/WINC-520 
```

Comment:
Nit: I would have preferred the TODO added in the commit where this change was introduced

Reply:
fixed